src/Symfony/Component/Validator/Constraints/SequentiallyValidator.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+
+/**
+ * @author Maxime Steinhausser <maxime.steinhausser@gmail.com>
+ */
+class SequentiallyValidator extends ConstraintValidator
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function validate($value, Constraint $constraint)
+    {
+        if (!$constraint instanceof Sequentially) {
+            throw new UnexpectedTypeException($constraint, Sequentially::class);
+        }
+
+        if (null === $value) {
```

Comment:
Need to allow passing value values. Otherwise following example won't be validated

```
@Assert\Sequentially({
  @Assert\NotNull(),
  @Assert\NotBlank()
})
```

Reply:
Of course you're right. Good catch, thanks